presidio-anonymizer/presidio_anonymizer/operators/keep.py

```diff
@@ -23,6 +24,26 @@ def operator_name(self) -> str:
         """Return operator name."""
         return "keep"
```

Comment:
Can we provide a different name to the keep deanonymizer? It might be confusing having two operators with the same name.

Reply:
How about?

```suggestion
        return "deanonymize_keep"
```

Reply:
Done! and fixed tests to check for `deanonymize_keep`. If you where thinking of other name just tell. It's a quick change